ebpf/disk-failure/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
Why re-copy the license here?

Reply:
I agree, I will removed this useless file.